Please investigate that test fail (can't be related to this PR I guess)

```
FAILED test_backups2datalad.py::test_1 - AssertionError: assert 1 == 0
===================
```

--- a/tools/chasseturls.py
+++ b/tools/chasseturls.py
@@ -10,8 +10,8 @@ def main(datasets):
     for ds in map(Dataset, datasets):
         ds.repo.always_commit = False
         changed = False
-        for path in ds.repo.get_annexed_files():
-            for url in ds.repo.get_urls(path):
+        for key, v in ds.repo.whereis(None, output="full", options=["--all"]).items():
+            for url in v.get("00000000-0000-0000-0000-000000000001", {}).get("urls"):
                 m = re.fullmatch(
                     r"https://(?P<domain>[^/]+)/api/dandisets/\d+/versions/"
                     r"draft/assets/(?P<asset_id>[^?/]+)/download/",
@@ -21,8 +21,8 @@ def main(datasets):
                     new_url = (
                         f"https://{m['domain']}/api/assets/{m['asset_id']}/download/"
                     )
-                    ds.repo.rm_url(path, url)
-                    ds.repo.add_url_to_file(path, new_url, batch=True)
+                    ds.repo.call_annex(["unregisterurl", key, url])
+                    ds.repo.call_annex(["registerurl", key, new_url])
                     changed = True
         if changed:
             ds.save(

--- a/tools/chasseturls.py
+++ b/tools/chasseturls.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 import re
 import click
 from datalad.api import Dataset

